jetty-deploy/src/main/java/org/eclipse/jetty/deploy/providers/ScanningAppProvider.java

```diff
@@ -310,15 +312,26 @@ public void setMonitoredDirectories(Collection<String> directories)
         }
     }
     
-    /* ------------------------------------------------------------ */
+
     protected void setRecursive(boolean recursive)
     {
         _recursive = recursive;
     }
 
-    /* ------------------------------------------------------------ */
+
     public void setScanInterval(int scanInterval)
     {
         _scanInterval = scanInterval;
     }
+
+
+    @ManagedOperation("Scan the monitored directories")
```

Comment:
Add `impact="ACTION"` to `@ManagedOperation`.

Reply:
done.